src/window/gui/Gui.cpp

```diff
@@ -459,14 +464,91 @@ void Gui::ImGuiWMNewFrame() {
     }
 }
 
+void Gui::ApplyResolutionChanges() {
+    ImVec2 size = ImGui::GetContentRegionAvail();
+
+    const float aspectRatioX = CVarGetFloat("gAdvancedResolution_aspectRatioX", 16.0f);
+    const float aspectRatioY = CVarGetFloat("gAdvancedResolution_aspectRatioY", 9.0f);
+    const uint32_t verticalPixelCount = CVarGetInteger("gAdvancedResolution_verticalPixelCount", 480);
+    const bool verticalResolutionToggle = CVarGetInteger("gAdvancedResolution_verticalResolutionToggle", (int)false);
```

Comment:
if you use `.` instead of `_` these will appear as
```json
{
  "gAdvancedResolution": {
    "aspectRatioX": "16.0f",
    "aspectRatioY": "16.0f",
    "verticalPixelCount": 480,
    "verticalResolutionToggle":0,
  }
}
```
insead of
```json
{
  "gAdvancedResolution_aspectRatioX": "16.0f",
  "gAdvancedResolution_aspectRatioY": "16.0f",
  "gAdvancedResolution_verticalPixelCount": 480,
  "gAdvancedResolution_verticalResolutionToggle":0,
}
```

Reply:
Ah! Thank you.